client/src/commands/editGatewayCommand.ts

```diff
@@ -305,19 +233,18 @@ async function editIdentity(gateway: FabricGatewayRegistryEntry, fabricGatewayRe
         // Ask for Connection Profile
         const result: string = await UserInputUtil.browseEdit('Enter a file path to a connection profile file', quickPickItems, openDialogOptions, gateway.name) as string;
         if (!result) {
-            // Connection Profile is needed to import identity so throw error if not given
-            throw new Error('Connection Profile required to import identity to file system wallet');
+            // Connection Profile is needed to import identity so cancel if not given
```

Comment:
is this still true now we always ask for the MSPid?
If not, can we ask for the connection profile after the identity? so that if the user clicks on the identity, they edit the identity first, then provide the connection profile - just like clicking on the other tree items